mocket/bytes.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import annotations
+
+from typing import Sequence
+
+from typing_extensions import Self
+
+from mocket.core.entry import MocketBaseEntry, MocketBaseRequest, MocketBaseResponse
+from mocket.core.mocket import Mocket
+from mocket.core.types import Address
+
+
+class MocketBytesRequest(MocketBaseRequest):
+    def __init__(self) -> None:
+        self._data = b""
+
+    @property
+    def data(self) -> bytes:
+        return self._data
+
+    @classmethod
+    def from_data(cls: type[Self], data: bytes) -> Self:
+        request = cls()
+        request._data = data
+        return request
+
+
+class MocketBytesResponse(MocketBaseResponse):
+    def __init__(self, data: bytes | str | bool) -> None:
```

Comment:
`MocketBytesResponse` can be created with `bytes`/`str`/`bool`? Isn't it confusing?